docs/source/cuda/cuda_array_interface.rst

```diff
@@ -104,3 +100,23 @@ Differences with CUDA Array Interface (Version 1)
 Versions 0 and 1 of the CUDA Array Interface neither clarified the
 **strides** attribute for C-contiguous arrays nor specified the treatment for
 zero-size arrays.
+
+
+Interoperability
+----------------
+
+The following Python libraries have adopted the CUDA Array Interface:
+
+- Numba
+- `CuPy <https://docs-cupy.chainer.org/en/stable/reference/interoperability.html>`_
+- `PyTorch <https://pytorch.org>`_
+- `PyArrow <https://arrow.apache.org/docs/python/generated/pyarrow.cuda.Context.html#pyarrow.cuda.Context.buffer_from_object>`_
+- `mpi4py <https://mpi4py.readthedocs.io/en/latest/overview.html#support-for-cuda-aware-mpi>`_
+- `ArrayViews <https://github.com/xnd-project/arrayviews>`_
+- The **RAPIDS** stack:
```

Comment:
Need this be bold font? it renders strangely.

Reply:
Ah...it's a leftover from previous style. Fixed.